### ISSUE TITLE
1.8.0.1 produces "TypeError: Cannot read property 'paused' of undefined"

### ISSUE THREAD

**Issue by KoenLav** [meteor/meteor#10358]
When deploying using mup on the meteord Docker image for Node.js 8.9.4 our app ends up being unreachable, producing the error:

UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 946): TypeError: Cannot read property 'paused' of undefined.

I narrowed down the usage of .paused to minimongo and to some of the Streamers in modules.js, so there's probably something going wrong there, but not sure what.

Any pointers?

**Comment by benjamn** [comment]
The `paused` property makes me suspect this might be due to https://github.com/meteor/meteor/pull/10309, and specifically [this line in the `webapp` package](https://github.com/meteor/meteor/blob/aecd530db6c2402c15c9112f6fad58fbb9fbeee5/packages/webapp/webapp_server.js#L403), which was recently release in `webapp@1.7.2` (independently of Meteor 1.8.0.1).

**Comment by KoenLav** [comment]
That seems likely to be related, as we updated this package.

This only happens on one project, with the only major structural difference being that it has android & iOS in the platforms files, I think.

**Comment by KoenLav** [comment]
It might actually be related to some other package we're using which asks Meteor to only build the modern bundle.

I'm investigating :)

**Comment by KoenLav** [comment]
I copied some bits from this package: https://github.com/qualialabs/one

To attempt to only build the legacy bundle when actually deploying. But to figure out whether we were deploying I relied on process.env.ROOT_URL being set, which isn't the case in our CI environment, which led to the legacy bundle not being produced for the build which ended up inside the container on our server, at which point process.env.ROOT_URL was set, causing Meteor to look for a legacy bundle which wasn't there.

I'm relying on both process.argv.includes('build') and process.env.ROOT_URL now (to make sure that whenever we are building instead of running, or when we have the ROOT_URL set, we get the legacy bundle).

So I guess this can be closed!
